docker/lib/utils.sh

```diff
@@ -0,0 +1,11 @@
+handle_container_error() {
+result=$?
+if [ $result -ne 0 ]
+then
+    echo
+    echo "Hint: If you're told the 'container is not running' you'll need to start it first:"
+    echo
+    echo "    % docker-compose up -d website"
+    exit $result
+fi
```

Comment:
Should this all be indented a level?

Reply:
Why not it's inside a block?

Reply:
Is it not normal to indent from the `{` on L1?

Reply:
Oh the whole fucntion, lol, I can do that also.